libs/Functions.php

```diff
@@ -534,10 +542,16 @@ public function post_type($icon=true){
 						$post_icon_img = '<i class="fas fa-thumbtack"></i>';
 					}
 				}
+			}else{
+				$post_icon_img = false;
```

Comment:
Should be able to set `$post_icon_img = false;` first before the `if` which would avoid the need for an `else` altogether.